### ISSUE TITLE
Accessibility: images in Dark mode

### ISSUE THREAD

**Issue by prowlett** [chirun-ncl/chirun#147]
I use a lot of tikz diagrams. These might look something like this in the web version:

![Screenshot from 2023-02-08 16-38-26](https://user-images.githubusercontent.com/5698564/217594122-1af557ad-4ded-4752-a4bc-1329771ec3a5.png)

If I switch to Pastel mode, this looks fine:

![Screenshot from 2023-02-08 16-38-33](https://user-images.githubusercontent.com/5698564/217594227-6857c0e4-42fb-4928-8880-cfb97ead4be1.png)

However, if I switch to Dark mode, the black lines in my diagram don't look so clear:

![Screenshot from 2023-02-08 16-38-40](https://user-images.githubusercontent.com/5698564/217594315-66dfa5fc-91af-4e46-92b4-7a4986cf3122.png)

Is there anything I can do? I don't suppose there is a simple fix the system can apply to change the colour in the images. Would it be better if they images were given a white background via CSS, or would this break dark mode? Or do you have any advice about how to create my diagrams so that they are better at dealing with this, without impacting accessibility in other ways? For example, I wonder if I should be using a default mid tone colour rather than black for diagrams like this, but would that make them less clear in other modes? I wondered if this this something you have thought about when creating the themes.

**Comment by christianp** [comment]
The new theme has a display option "invert images in dark mode". When ticked, all images have their colours inverted automatically.

Here's an example:

![image](https://github.com/chirun-ncl/chirun/assets/19513/849daa92-4139-4814-b5b0-536e59d7a2c5)

![image](https://github.com/chirun-ncl/chirun/assets/19513/d81486dc-d8f2-46ba-936c-62ec171ea60e)

It makes photos look really weird, which is why it's an option and not always on.
